src/content/add-to-app/ios/project-setup.md

```diff
@@ -5,75 +5,71 @@ description: Learn how to integrate a Flutter module into your existing iOS proj
 ---
 
 Flutter UI components can be incrementally added into your existing iOS
-application as embedded frameworks. There are a few ways to embed Flutter 
-in your existing application.
-
-1. **Use the CocoaPods dependency manager and installed Flutter SDK.**
-  In this case, the `flutter_module` is compiled from
-  the source each time the app is built. (Recommended.)
-
-1. **Create frameworks for the Flutter engine, your compiled Dart code,
-  and all Flutter plugins.** Here, you manually embed the frameworks,
-  and update your existing application's build settings in Xcode.
-  This can be useful for teams that don't want to require every developer
-  to have the Flutter SDK and Cocoapods installed locally.
-
-1. **Create frameworks for your compiled Dart code,
-  and all Flutter plugins. Use CocoaPods for the Flutter engine.** 
-  With this option, embed the frameworks for your application
-  and the plugins in Xcode, but distribute the
-  Flutter engine as a CocoaPods podspec.
-  This is similar to the second option, but it provides
-  an alternative to distributing the large Flutter.xcframework.
-
-For examples using an app built with UIKit, 
-see the iOS directories in the [add_to_app code samples][]. 
-For an example using SwiftUI, see the iOS directory in [News Feed App][].
+application as embedded frameworks.
+To embed Flutter in your existing application,
+consider one of the following three methods.
+
+**Use the CocoaPods dependency manager and installed Flutter SDK.**
+: Each time you build the app,
+  it compiles the `flutter_module` from source.
+  (Recommended.)
+
+**Create frameworks for each component.**
+: Create frameworks for the Flutter engine, your compiled Dart code,
+  and all Flutter plugins in Xcode.
+  Embed these frameworks and update your existing application's build settings.
+  This method works for teams that don't want to require every developer
+  to install the Flutter SDK and Cocoapods on their local machines.
+
+**Create frameworks for Dart code and Flutter plugins. Use CocoaPods for the Flutter engine.**
+: Embed the frameworks for your iOS app and the plugins in Xcode,
+  but distribute the Flutter engine as a CocoaPods podspec.
+  This works like the second method,
+  but it provides an alternative to distributing the large Flutter.xcframework.
```

Comment:
Option B and C have a drawback that isn't listed here: https://github.com/flutter/flutter/issues/146957#issuecomment-2066841728

> The big limitation that Flutter has with `xcframework`s and CocoaPods is the inability to handle common dependencies, https://github.com/flutter/flutter/issues/130220. If both host app and Flutter module (Flutter module's plugin) define the same pod dependency, and Flutter module is integrated using Option B (with xcframeworks), there will be an errors like `Multiple commands produce 'CommonDependency.framework'`. The workaround is to link every plugin source (podspec) from the Flutter module directly to the host app (in Podfile), instead of using plugins' `xcframework`s produced by `build ios-framework` command, https://github.com/flutter/flutter/issues/114692. That was one of the reasons `build ios-framework --no-plugins` flag was introduced.

Should we add this `--no-plugins` flag as option D?